src/main/java/edu/kis/powp/jobs2d/command/history/CommandHistory.java

```diff
@@ -0,0 +1,27 @@
+package edu.kis.powp.jobs2d.command.history;
+
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+import javax.swing.DefaultListModel;
+
+import edu.kis.powp.jobs2d.command.DriverCommand;
+
+public class CommandHistory {
+
+    private static DefaultListModel<HistoryEntry> entryHistoryList = new DefaultListModel<>();
```

Comment:
a declaration should be limited to the interface, i.e. ListModel.

Reply:
It can not be like that. ListModel does not contains addElement method, which we are using.

Reply:
ok